sanic/request.py

```diff
@@ -302,24 +303,29 @@ def parse_multipart_form(body, boundary):
                 form_line[colon_index + 2:])
 
             if form_header_field == 'content-disposition':
-                if 'filename' in form_parameters:
-                    file_name = form_parameters['filename']
+                file_name = form_parameters.get('filename')
                 field_name = form_parameters.get('name')
             elif form_header_field == 'content-type':
-                file_type = form_header_value
-
-        post_data = form_part[line_index:-4]
-        if file_name or file_type:
-            file = File(type=file_type, name=file_name, body=post_data)
-            if field_name in files:
-                files[field_name].append(file)
+                content_type = form_header_value
+                content_charset = form_parameters.get('charset', 'utf-8')
+
+        if field_name:
+            post_data = form_part[line_index:-4]
+            if file_name:
+                form_file = \
```

Comment:
Please change this to use newlines (see my comment outside of here, because it won't let me put newlines in this comment)